docs/extended_api/pipelines/pipeline_producer_commit.md

```diff
@@ -0,0 +1,61 @@
+---
+grand_parent: Extended API
+parent: Pipelines
+---
+
+# `cuda::pipeline_producer_commit`
+
+Defined in header `<cuda/pipeline>`:
+
+```cuda
+template <cuda::thread_scope Scope>
+__host__ __device__
+void cuda::pipeline_producer_commit(cuda::pipeline<cuda::thread_scope_thread>& pipe,
+                                    cuda::barrier<Scope>& bar);
+```
+
+Binds operations previously issued by the current thread to the named
+  `cuda::barrier` such that a `cuda::barrier::arrive` is performed on completion.
+The bind operation implicitly increments the barrier's current phase to account
+  for the subsequent `cuda::barrier::arrive`, resulting in a net change of 0.
+
+## Parameters
+
+| `pipe` | The thread-scoped `cuda::pipeline` object to wait on. |
+| `bar`  | The `cuda::barrier` to arrive on.                      |
+
+## Notes
```

Comment:
This needs to be changed to the `Expects` form to be consistent with the rest of the documentation.